README.md

```diff
@@ -18,21 +18,21 @@ Inclue or extend our colour configuration and add our components to Tailwind's c
 
 ```js
 // tailwind.config.js
-import colors from '@nearform/quantum/dist/colors'
 
 module.exports = {
-    content: [
-        "./node_modules/@nearform/quantum/**/*.js"
-    ],
-    theme: {
-        colors,
-    }
-};
+  content: [
+    //your content files...
+    './node_modules/@nearform/quantum/**/*.js'
```

Comment:
Is this required? I would expect only the plugins sections below to be used

Reply:
Unfortunately so. Please read my PR description for further info :D

Reply:
as per the comment you received on the issue you opened on the tailwind repo, let's export a function that does this, instead of asking the user to write this line

Reply:
What if someone doesn't have node_modules at the same directory as their tailwind config? @simoneb

Reply:
would it be fair to assume that it's the same folder as the tailwind config file that the function is imported and executed from?

Reply:
I think on average this will be the case. But we shouldn't limit the scope of this package to just those applications should we?

Reply:
no we shouldn't, therefore we could do that by default, and we could accept an optional argument in that function which allows passing in e.g. the path of the tailwind config file that is executing the function, so we could resolve the path correctly

Reply:
Found a nice way to do it. Now it's no longer necessary to add in node_modules to content.